src/StudyRegistration/VLMDSubmission/CDESubmission.tsx

```diff
@@ -0,0 +1,302 @@
+            label='Core CDEs'
+          >
+            <Checkbox.Group style={{ width: '100%' }}>
+              <Row>
+                {(cdeInfoFromMDS.filter((entry) => entry.isCoreCDE).map((entry, i) => (
+                  <Col span={12} key={i}>
+                    <Checkbox value={entry.option}>{entry.option}</Checkbox>
+                  </Col>
+                )))}
+              </Row>
+            </Checkbox.Group>
+          </Form.Item>
+          <Form.Item
+            name='selectedCDEs'
+            label='All CDEs'
+            extra='Search and select from all CDEs'
+          >
+            <Select
+              mode='multiple'
+              allowClear
+              style={{ width: '100%' }}
+              options={cdeInfoFromMDS.map((entry) => ({ label: entry.option, value: entry.option }))}
+            />
+          </Form.Item>
+          <Divider plain>Administration
+            <Tooltip title='This information will be used to contact you regarding your submission status. This information is not stored on the HEAL Data Platform'>
+              <QuestionCircleOutlined className='study-reg-form-item__middle-icon' />
+            </Tooltip>
+          </Divider>
+          <Form.Item
+            name='First Name'
+            label='Submitter First Name'
+            rules={[
+              {
+                required: true,
+              },
+            ]}
+          >
+            <Input />
+          </Form.Item>
+          <Form.Item
+            name='Last Name'
+            label='Submitter Last Name'
+            rules={[
+              {
+                required: true,
+              },
+            ]}
+          >
+            <Input />
+          </Form.Item>
+          <Form.Item
+            name='E-mail Address'
+            label='E-mail Address'
+            rules={[
+              {
+                type: 'email',
+                message: 'The input is not valid E-mail',
+              },
+              {
+                required: true,
+              },
+            ]}
+          >
+            <Input />
+          </Form.Item>
+          <Form.Item {...TAIL_LAYOUT}>
+            <Space>
+              {(!props.userHasAccessToSubmit) ? (
+                <Tooltip title={'You don\'t have permission to submit CDEs'}>
+                  <Button type='primary' disabled>
+                    Submit CDEs
+                  </Button>
+                </Tooltip>
+              ) : (
+                <Button type='primary' onClick={onSubmitButtonClick} disabled={props.disableCDESubmissionForm || cdeFormSubmitting} loading={cdeFormSubmitting}>
+                  Submit CDEs
+                </Button>
+              )}
+            </Space>
+          </Form.Item>
+          { (studyRegistrationConfig.cdeSubmissionDisclaimerField)
+              && (
+                <Typography className='study-reg-disclaimer-text'>
+                  {parse(studyRegistrationConfig.cdeSubmissionDisclaimerField)}
```

Comment:
Yeah good point. It would be good to have some sanitations, but you are right I think the risk is minimal since we controls what can goes into portal config